sys/cbor/cbor.c

```diff
@@ -236,12 +233,12 @@ void dump_memory(const unsigned char *data, size_t size)
         return;
     }
 
-    DEBUG("0x");
+    printf("0x");
```

Comment:
I don't completely understand why a function [cbor_stream_decode](https://github.com/OTAkeys/RIOT/blob/9d503966c437fb4d05ab627de01857bef9248779/sys/include/cbor.h#L212) in cbor.h header shouldn't print anything when `enable_debug` is 0? From my point of view if you don't want to print the stream just don't call the print function from the other modules/apps :-).

The unit test prints the stream all the time because it uses module `cbor_print` and  [my_cbor_print](https://github.com/OTAkeys/RIOT/blob/9d503966c437fb4d05ab627de01857bef9248779/tests/unittests/tests-cbor/tests-cbor.c#L32)